xml/gpu_passthru.xml

```diff
@@ -0,0 +1,503 @@
+   <para>
+    For &mswin; guests, we recommend disabling MSR (model-specific register) to
+    avoid crash of the guest. Create the file
+    <filename>/etc/modprobe.d/kvm.conf</filename> and add the following
+    content:
+   </para>
+<screen>options kvm ignore_msrs=1</screen>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-ovmf">
+   <title>Install and enable UEFI firmware</title>
+   <para>
+    For proper &gpuback; functionality, the host need to boot using the UEFI
+    firmware.
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Install the <package>qemu-ovmf</package> package that includes UEFI
+      firmware images:
+     </para>
+<screen>&prompt.sudo;zypper install qemu-ovmf</screen>
+    </step>
+    <step>
+     <para>
+      Get the list of OVMF <literal>bin</literal> and <literal>vars</literal>
+      files by filtering the results of the following command:
+     </para>
+<screen>&prompt.user;rpm -ql qemu-ovmf</screen>
+    </step>
+    <step>
+     <para>
+      Enable OVMF in the &libvirt; &qemu; configuration in the file
+      <filename>/etc/libvirt/qemu.conf</filename> by using the list obtained
+      from the previous step. It should look similar to the following:
+     </para>
+<screen>
+nvram = [
+"/usr/share/qemu/ovmf-x86_64-4m.bin:/usr/share/qemu/ovmf-x86_64-4m-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-4m-code.bin:/usr/share/qemu/ovmf-x86_64-4m-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-smm-ms-code.bin:/usr/share/qemu/ovmf-x86_64-smm-ms-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-smm-opensuse-code.bin:/usr/share/qemu/ovmf-x86_64-smm-opensuse-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-ms-4m-code.bin:/usr/share/qemu/ovmf-x86_64-ms-4m-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-smm-suse-code.bin:/usr/share/qemu/ovmf-x86_64-smm-suse-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-ms-code.bin:/usr/share/qemu/ovmf-x86_64-ms-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-smm-code.bin:/usr/share/qemu/ovmf-x86_64-smm-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-opensuse-4m-code.bin:/usr/share/qemu/ovmf-x86_64-opensuse-4m-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-suse-4m-code.bin:/usr/share/qemu/ovmf-x86_64-suse-4m-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-suse-code.bin:/usr/share/qemu/ovmf-x86_64-suse-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-opensuse-code.bin:/usr/share/qemu/ovmf-x86_64-opensuse-vars.bin",
+"/usr/share/qemu/ovmf-x86_64-code.bin:/usr/share/qemu/ovmf-x86_64-vars.bin",
+]
+</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-reboot">
+   <title>Reboot the host machine</title>
+   <para>
+    For most of the changes in the above steps to take effect, you need to
+    reboot the host machine:
+   </para>
+<screen>&prompt.sudo;shutdown -r now</screen>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="gpu-passthru-guest">
+  <title>Configuring the guest</title>
+
+  <para>
+   This section describes how to configure the guest virtual machine so that it
+   can use the host's GPU &nvidia; graphic card. Use &vmm; or
```

Comment:
```suggestion
   can use the host's &nvidia; GPU. Use &vmm; or
```

Reply:
As the earlier instructions in the chapter made clear, the GPU for pass-through is **not** to be the default graphics card, i.e. the card displaying the host's OS screen. It must be a dedicated GPU.

AIUI the Tesla series do not actually have monitor outputs and so cannot be used to display graphics. They are dedicated GPGPUs and not graphics cards.